### PR TITLE
Update to 0.4.1

### PR DESCRIPTION
**Pull request by Tworley96**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = magql
-version = 0.4.0
+version = 0.5.0
```
Comment (davidism) [inline setup.cfg:3]: 0.4.1?

Comment (Tworley96) [reply setup.cfg:3]: I pushed to pypi with 0.4.1, but I thought we were going with the version that was one ahead of what we were at now. Like how we had 0.4.0 while we working on developing it.

Comment (davidism) [reply setup.cfg:3]: Ah, I thought this was the 0.4.1 release because you were adding the changelog.

Comment (Tworley96) [reply setup.cfg:3]: It is supposed to be the 0.4.1 release. I didn't bump the version to 0.5.0 in the 0.4.0 release but should have. This is making up for that. I think this is all correct.

**Pull request merged**